docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy:alpine
```

Comment:
Not sure I agree with using jwilder/nginx-proxy. It seems to work in quite non-standard way of reading docker containers and automatically exposing them. While this is nice, it is slightly unexpected behaviour, and can take while to understand how it works.

Reply:
Can you elaborate on the unexpected behavior you are experiencing? Spinning up these services together automatically puts them in the same network, thereby exposing them to one another. [links](https://docs.docker.com/compose/compose-file/#links) also connect the containers in a way that is natural to docker. 

I did not add any comments to this docker-compose file, I can add some here to give some more context on what is going on.

Reply:
One other thing that I like about this image is that it allows you to set up as many proxies as you like. You can even move the proxy up a layer in your dev environment and have it running on its own and whenever you have a project you need a proxy for then you can add it to the network and proxy as many projects through it at once. For the purposes of PWA that means that you can run multiple PWA storefronts through the proxy, essentially have as many PWA domains as you like that all have their own service workers assigned and running without collision because they are running in different containers and have different domains, etc.